browser/main/NoteList/index.js

```diff
@@ -543,6 +548,44 @@ class NoteList extends React.Component {
     this.setState({ selectedNoteKeys: [] })
   }
 
+  cloneNote () {
+    const { selectedNoteKeys } = this.state
+    const { dispatch, location } = this.props
+    const { router } = this.context
```

Comment:
please remove `router`. It is never used.

Reply:
resolved